app/presenters/vendor_api/interview_presenter.rb

```diff
@@ -0,0 +1,32 @@
+class VendorAPI::InterviewPresenter < VendorAPI::Base
+  attr_reader :interview
+
+  def initialize(version, interview)
+    super(version)
+    @interview = interview
+  end
+
+  def as_json
+    schema.to_json
+  end
+
+  def schema
+    {
+      id: interview.id.to_s,
+      provider_code: provider_code(interview),
+      date_and_time: interview.date_and_time.iso8601,
+      location: interview.location,
+      additional_details: interview.additional_details,
+      cancelled_at: interview.cancelled_at&.iso8601,
+      cancellation_reason: interview.cancellation_reason,
+      created_at: interview.created_at.iso8601,
+      updated_at: interview.updated_at.iso8601,
+    }
+  end
+
+private
+
+  def provider_code(interview)
+    Provider.find(interview.provider_id).code
```

Comment:
wondering around having to do this query here, any chance this can be passed in to avoid a single query on each interview if this is retrieved from GET applications 🤔 ?

Reply:
Not really, provider codes can vary a great deal, especially through the ratifying provider relationship.

Reply:
Ah, may be able to just do `interview.provider.code`

Reply:
See also: https://github.com/DFE-Digital/apply-for-teacher-training/pull/6429